src/Sylius/Bundle/ResourceBundle/Model/ResourceLogEntry.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Model;
+
+use Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry;
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+abstract class ResourceLogEntry extends AbstractLogEntry implements ResourceInterface
```

Comment:
There is different gedmo mapping for *orm* and *odm* so I don't think it's a big problem, especially that we don't really support *odm* for now very well 😄